docs/userstories.rst

```diff
@@ -35,3 +35,73 @@ During the development, it becomes clear that a major part of the implemented
+
+.. _bamgroup:
+
+BAM Group (structural reliability)
+----------------------------------
+The objective is to guarantee the safety of a construction by determining the failure probability.
+This is done by repeated evaluation of a structural model in a probabilistic approach (*adaptive importance sampling*).
+Herein, an approximation of the probability of failure is computed from a joint distribution function of the
+material parameters. Usually within this framework a large number of evaluations of the structural model are required which is
+why the replacement of the structural model by a surrogate / *reduced order model* is necessary to increase efficiency.
+
+Another aspect of the probabilistic approach is the calibration of the model by means of *Bayesian Inference* in which
+the distribution function is determined based on the prior knowledge and the likelihood function. The likelihood
+describes the probability that the numerical model of the lab experiment reproduces the experimental data for given model parameters.
+Therefore, the numerical model has to be evaluated for various parameter values and the experimental data has to be processed. Here, for each of the different lab experiments a model and corresponding *metadata* should exist, such that these
+are evaluated collectively. At the same time a query of the *database* should extract the relevant metadata and process the raw
+data accordingly.
+
+With the above example we would like to address workflows which consist of several modules, where each module
+requires a different expertise (FEM, ROM, Bayesian Inference, ...), leading to the implementation and development of the
+workflow being a group effort.
+
+Collaborators taking part in the presented workflow could be grouped by *developers* and *users*.
+
+Developers
+^^^^^^^^^^
+It is important that all modules (heterogeneous models) are embedded into a common framework and that the 
+*relations and dependencies* between modules should be defined with a *portable data structure*.
+For collaboration with others within the group automation of the installation of potentially complex
+software environments (up to *machine-independent execution*) within the workflow tool is thought to be a great but
+not compulsory feature.
+The *reusability* of each module in several workflows is a basic requirement to avoid duplication of work (code).
+Another fundamental feature for effective teamwork is a comprehensive *documentation* of the API of the workflow tool as well as
+documentation of the implemented workflow (automatically generated).
+The role of *metadata* as described above is twofold. On the one hand, for the compute environment software packages and
+versions used (preferably automatically handled by the workflow system) need to be documented. On the other hand, metadata should 
+be provided to describe and process experimental data in an automated way. In general metadata and data should comply with
+the FAIR principles.
+
```

Comment:
I'm somehow missing the CI integration with regular updates of dependent packages (like a new numpy version, ...) and the documentation, which combinations are currently (or have been in the past) working.

Reply:
In addition, sharing the workflow through a plattform might be nice, so the automatically generated meta data is published with the workflow and searchable.